### PR TITLE
Stress requirements for clean system

### PR DESCRIPTION
**Pull request by LindsayHill**
More strongly stress the requirement for a **clean** system for using the one-line install.

Fixed a couple of minor language things. 

Addresses https://github.com/StackStorm/st2docs/issues/368

/cc @emedvedev

### CODE REVIEW

File: docs/source/install/index.rst [hunk 1]
```diff
@@ -10,10 +10,21 @@ this command:
 
    curl -sSL https://stackstorm.com/packages/install.sh | bash -s -- --user=st2admin --password='Ch@ngeMe'
 
+.. danger::
+
+  The installation script is an opinionated installation of |st2|. It assumes that you have a clean, basic
+  installation of Ubuntu or RHEL/CentOS, similar to what you get with a basic installation from ISO. If
+  you are trying to install |st2| on a server with many existing services, or maybe a 'pre-loved' VM that
```
Comment (emedvedev) [inline docs/source/install/index.rst]: > If you are trying to install ... that you will run into problems.

Looks like there's a typo or an unfinished edit here.

**Comment by emedvedev** [comment]
:heart:

File: docs/source/install/index.rst (hunk 1 above)
Comment (LindsayHill) [inline docs/source/install/index.rst]: Whoops, there is indeed. Mobile now, will fix it tomorrow

File: docs/source/install/index.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Installation
 ============
 
-That's OK! You're busy, we get it. How do you just get started? Get yourself a clean 64-bit Linux
+That's OK! You're busy, we get it. How do you just get started? Get yourself a **clean** 64-bit Linux
```
Comment (armab) [inline docs/source/install/index.rst]: !!! :)

**Comment by Kami** [comment]
LGTM :+1: 

Maybe also not a bad idea to explicitly point out that if for some reason installer script fails half way through, user should re-run it on a new clean server.

**Comment by LindsayHill** [comment]
Good point @Kami will add another update about that

**Pull request merged**